CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,50 @@
 # Changelog
 
+## [1.58.125](https://github.com/brave/brave-browser/releases/tag/v1.58.125)
+
+### Web3
+
+ - Added Filecoin to Brave Wallet. ([#30342](https://github.com/brave/brave-browser/issues/30342))
+ - Added support for EIP-6963. ([#30595](https://github.com/brave/brave-browser/issues/30595))
+ - Added Solana to the top of the network list. ([#31506](https://github.com/brave/brave-browser/issues/31506))
```

Comment:
Should be Moved Solana instead of Added. It was already there but not on top but had regressed it before.

Reply:
Done.